nntrainer/layers/conv2d_layer.cpp

```diff
@@ -113,7 +114,7 @@ static void col2im(const Tensor &col_matrix, const TensorDim &kdim,
  * @param[in] padding padding information
  * @param[in] mstride stride value : x, y direction
  * @param[in] dilation kernel dilation factor : x, y each
- * @param[in] channel_mode loop with channel first
+ * @param[in] channel_mode loop with channel first (not used)
```

Comment:
If it is not used, how about delete it?

Reply:
I'll delete this and leave the part commented as @kparichay suggested.

Reply:
This will be done later